src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
```

Comment:
The stylesheet link references a '.scss' file. Ensure that this Sass file is compiled to CSS and the correct CSS file path is used here. Browsers cannot directly interpret Sass files.